s3_storage_provider.py

```diff
@@ -62,7 +62,7 @@ class S3StorageProviderBackend(StorageProvider):
     def __init__(self, hs, config):
         self.cache_directory = hs.config.media.media_store_path
         self.bucket = config["bucket"]
-        self.storage_class = config["storage_class"]
+        self.eargs = config["eargs"]
```

Comment:
Can we unabbreviate `eargs` to `extra_args` please?
It's not clear what the `e` stands for unless you stumble upon the call to `upload_file`.